freqtrade/rpc/telegram.py

```diff
@@ -636,13 +636,15 @@ def _show_config(self, update: Update, context: CallbackContext) -> None:
 
         else:
             sl_info = f"*Stoploss:* `{val['stoploss']}`\n"
-
+        import json
         self._send_msg(
             f"*Mode:* `{'Dry-run' if val['dry_run'] else 'Live'}`\n"
             f"*Exchange:* `{val['exchange']}`\n"
             f"*Stake per trade:* `{val['stake_amount']} {val['stake_currency']}`\n"
             f"*Max open Trades:* `{val['max_open_trades']}`\n"
             f"*Minimum ROI:* `{val['minimal_roi']}`\n"
+            f"*Ask strategy:* ```\n{val['ask_strategy']}```\n"
```

Comment:
`json.dumps` here as well?

Reply:
I'm not sure -  you're right, it must be aligned between the two sides - but if you look at the screenshot - the only difference is the quotes (and False or false) ...

so explicit json conversation might not be necessary ... ?